### PR TITLE
How expressions are evaluated on the Python script.

### PR DESCRIPTION
**Pull request by RuthAtem**
To display the evaluation of an expression on the Python script, the print function must be used else nothing will be shown.

### CODE REVIEW

**Comment by ch3nkula** [comment]
@RuthAtem, I was expecting to see the script as well, I see only the captured info on the CMD prompt. Can you include the script for this purpose with the name called `display_eval.py`

File: Ruth/Chapter_1/exercise_1.12.6 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Python 2.7.12 (default, Dec  4 2017, 14:50:18) 
+[GCC 5.4.0 20160609] on linux2
+Type "copyright", "credits" or "license()" for more information.
+>>> 6 + 4 * 9
+42
+>>> 
+==================== RESTART: /home/riri/exercise_1.12.6 ====================
```
Comment (ch3nkula) [inline Ruth/Chapter_1/exercise_1.12.6]: Write a script to demonstrate this as stated, I know you had to change content to show both but I need to see the script. The name of the script should be `display_eval.py`. :)

**Comment by ch3nkula** [comment]
The script can be both lines of code for demonstration purposes! I mean something like this;
```
6 + 9 * 4 # will not display the output on the prompt

print(6 + 9 * 4) # will display output on the prompt
```

Name of the script should be `display_eval.py`

File: Ruth/Chapter_1/display_eval.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+(6 + 4 * 9) # this will not display the output on the prompt
```
Comment (ch3nkula) [inline Ruth/Chapter_1/display_eval.py]: Is this line suppose to be `6 + 4 * 9` or `(6 + 4 * 9)`? Maybe you need to fix this?

**Comment by RuthAtem** [comment]
Ok Thank you!

**Pull request merged**